### ISSUE TITLE
Policy Table update sequence

### ISSUE THREAD

**Issue by justinjdickow** [smartdevicelink/sdl_server#13]
>On Step 8, it is explained that "Typically Usage and Error Counts, Device Data, and Module Meta contain data to be recored." But on Step 5, it is said that "The mobile library will ignore the request body containing the policy table snapshot". Who is supposed to send or add "Device Data" to the server if the mobile library does not touch the policy table snapshot? Or Error Counts, Device Data and Module Meta are supposed to be sent to the server with the policy table snapshot as separate data?

**Comment by ssmereka** [comment]
**TL;DR**
Only SDL core populates this data which is then sent to the SDL server.

SDL core populates its local policy table with information including Error Counts, Device Data, and Module Meta information.  This information could come from anywhere, but it is important that core records it into the policy table.  When it comes time for a policy table update the aforementioned fields are then added to a policy table snapshot, symmetrically encrypted, then jammed into the body portion of an proprietary OnSystemRequest RPC.  This information is sent to a mobile device (with an SDL library) to be forwarded to the SDL server.

The mobile device does not directly report data to the SDL server.
